sdk/eventhub/azure-eventhub/stress/scripts/azure_eventhub_consumer_stress_async.py

```diff
@@ -92,7 +92,7 @@ def parse_starting_position(args):
 
 args = parser.parse_args()
 starting_position = parse_starting_position(args)
-LOGGER = get_logger(args.log_filename, "stress_receive_async", level=logging.DEBUG, print_console=args.print_console)
+LOGGER = get_logger(args.log_filename, "stress_receive_async", level=logging.ERROR, print_console=args.print_console)
```

Comment:
is there a way to pass in the log level in the args ( similar to transport type ) ? This way we can default to `logging.ERROR` if not present but can switch it up easily when we want debug level.

Reply:
Yes there is, let me do that!